src/PSR7/Validators/BodyValidator/BodyValidator.php

```diff
@@ -32,7 +35,11 @@ public function __construct(SpecFinder $finder)
         $this->finder = $finder;
     }
 
-    /** {@inheritdoc} */
+    /**
+     * @throws InvalidBody
```

Comment:
Well, no one will see these exceptions if they utilize interface instead of exact class, so more common `ValidationFailed` provided in `inheritdoc` is better